src/huggingface_hub/inference/_client.py

```diff
@@ -2392,7 +2392,13 @@ def visual_question_answering(
         return VisualQuestionAnsweringOutputElement.parse_obj_as_list(response)
 
     def zero_shot_classification(
-        self, text: str, labels: List[str], *, multi_label: bool = False, model: Optional[str] = None
+        self,
+        text: str,
+        labels: List[str],
+        *,
+        multi_label: bool = False,
+        model: Optional[str] = None,
+        hypothesis_template: str = "This example is {}.",
```

Comment:
Could you invert those two. By convention, I put `model` last in all signatures (since not specific for the method)